cfgov/v1/atomic_elements/organisms.py

```diff
@@ -911,6 +911,13 @@ class SnippetList(blocks.StructBlock):
     heading = blocks.CharBlock(required=False)
     body = blocks.RichTextBlock(required=False)
     image = atoms.ImageBasic(required=False)
+    actions_column_width = blocks.CharBlock(
+        label='Actions col width (%)',
```

Comment:
Let's expand this out to `Actions column width (%)`

Reply:
Is the actions column always the last column? If so, it may be clearer to say "Last column width (%)"

Reply:
We can't for the first suggestion, it breaks to a new line on the `(%):`. I can try the second suggestion.

Reply:
Remembered that there's interest in having more than two columns. To avoid any issues down the road I'm going to keep this as "Actions column"

Reply:
> We can't for the first suggestion, it breaks to a new line on the (%):.

That's going to depend on the window width, so either way it could break like that. I would avoid abbreviations in the labels.